assets/js/components/Stepper/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Stepper component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Children, cloneElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Tick from '../../../svg/icons/tick.svg';
+import { STEP_STATUS } from './constants';
+
+// English-language names for the step statuses.
+const STEP_STATUS_NAMES = {
+	[ STEP_STATUS.UPCOMING ]: 'upcoming',
+	[ STEP_STATUS.ACTIVE ]: 'active',
+	[ STEP_STATUS.COMPLETED ]: 'completed',
+};
```

Comment:
It seems the values here should be translated since the values are used in a user-facing string.

Reply:
Thanks, good spot - this also ties in with your comment below, about having a single string per status, which makes sense too.